Makefile

```diff
@@ -81,6 +81,12 @@ test-style: vendor $(GOLANGCI_LINT)
 	$(GOLANGCI_LINT) run
 	@scripts/validate-license.sh
 
+.PHONY: test-completion
+test-completion: TARGETS = linux/amd64
```

Comment:
Any chance these tests could be made to work on multiple platforms?

Reply:
I made the assumption that the person running these tests would run docker under linux/amd64.  I can try to figure out where docker is being run and choose the correct platform.

Is that what you mean?